go-controller/pkg/kubevirt/types.go

```diff
@@ -0,0 +1,5 @@
+package kubevirt
```

Comment:
Since ovn-k has the concept of node ip migration maybe we can renaming the feature here to "pod ip migration" and implement the following as kubevirt specific:
- Discover the pod network info. 
- Discover when to switch nodes at point to point routing.